tensai-fabric/src/main/resources/tensai.mixins.json

```
@@ -3,7 +3,7 @@
   "package": "dev.vmsa.tensai.fabric.mixins",
```

I think you should change the ``package`` field instead of putting entire path in ``mixins`` array.

```suggestion
  "package": "dev.phomc.tensai.fabric.mixins",
```

---

intellij prob